wpt-metadata/mapper/headless/webdriver/tests/bidi/browsing_context/navigate/error.py.ini

```diff
@@ -1,3 +1,6 @@
 [error.py]
   [test_with_new_navigation_inside_page]
-    expected: FAIL
+    expected: [FAIL, PASS]
```

Comment:
what's the reason for flakiness?

Reply:
https://github.com/GoogleChromeLabs/chromium-bidi/issues/2568